template/web/src/utils/paymasterClient.ts

```diff
@@ -1,10 +1,15 @@
 import { ENTRYPOINT_ADDRESS_V06 } from 'permissionless';
 import { paymasterActionsEip7677 } from 'permissionless/experimental';
-import { createClient, http } from 'viem';
+import { createClient, http, createPublicClient } from 'viem';
 import { baseSepolia } from 'viem/chains';
 
 const paymasterService = process.env.NEXT_PUBLIC_PAYMASTER_URL ?? '';
 
+export const client = createPublicClient({
+  chain: baseSepolia,
+  transport: http(),
+});
```

Comment:
This is used as the client to pass into `isWalletASmartWallet`

Reply:
Had to use `client` rather than `paymasterClient` because `client` is created using `createPublicClient` and is of type `PublicClient`, which is what `isWalletASmartWallet` expects.

Maybe we should update `isWalletASmartWallet` to use `createClient` rather than `createPublicClient` from Viem?